tests/Doctrine/Tests/DBAL/Functional/Driver/PDOPgSql/DriverTest.php

```diff
@@ -21,6 +23,43 @@ protected function setUp()
     }
 
     /**
+    @dataProvider getDatabaseParameter
```

Comment:
@kimhemsoe this looks like borken indentation. Can you fix this? After that I'll merge.